superdesk/resource_fields.py

```diff
@@ -22,5 +23,6 @@
 ITEMS = "_items"
 LINKS = "_links"
 ETAG = "_etag"
-VERSION = "_current_version"
+VERSION = CURRENT_VERSION = "_current_version"
+LATEST_VERSION = "_latest_version"
```

Comment:
I wonder what's the usecase for this field, seems like something that would require an update to all previous versions on each save, but at the end it's just duplicating the `_current_version` value from the main collection

Reply:
It's populated on fetch (so no need to update previous items), and is also used in the front-end.

Reply:
when fetching previous versions?

Reply:
Correct